optaweb-employee-rostering-frontend/public/assets/translations/en/ConnectionStatus.yaml

```diff
@@ -0,0 +1,3 @@
+cannotConnect: Cannot Connect
+cannotConnectMsg:
+  Please check your network connection. When connection is available the application will be functional again.
```

Comment:
I wouldn't make those kinds of promises. :-)

My suggestion: "No connection to server"
"Connection to the planning backend has been lost. Please check your network connection."

Reply:
Copied from https://github.com/kiegroup/optaweb-vehicle-routing/blob/039716a4ee333144abac9286523dbd1460ffa832/optaweb-vehicle-routing-frontend/src/ui/connection/ConnectionManager.tsx#L55,

I can reword it

Reply:
The "When connection is available the application will be functional again." is meant to inform the user that if the backend comes online, it will automatically reconnect. Although we might not want to promise "functional".